<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BidsAssistantClass</class>
 <widget class="QMainWindow" name="BidsAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>697</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BidsAssistant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Resources/reset2.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>新增</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Resources/add.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_query">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Resources/search.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_export">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Resources/excel.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_checkstatus">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>服务状态</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Resources/status.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tv_tabledata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1331</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>244</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QComboBox" name="cb_gczt">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="le_price_begin">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_price_end">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QDateEdit" name="dateedit_zb_start">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateedit_zb_end">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QComboBox" name="cb_ptyj">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>93</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>工程状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>中标价（万元）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>中标时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>平台业绩</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="le_companyname">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QComboBox" name="cb_qiyezizhi">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QComboBox" name="cb_gclb">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="le_projectname">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>763</x>
       <y>30</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="le_projectmgr_name">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>178</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="cb_zjqk">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QComboBox" name="cb_zc">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QComboBox" name="cb_jzs">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>93</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>项目经理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>在建情况</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>职称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>建造师</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>企业名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>企业资质</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>工程类别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>工程名称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>30</y>
       <width>93</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>面积（平方米）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>竣工时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>验收时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>30</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLineEdit" name="le_mianji_begin">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_mianji_end">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QDateEdit" name="dateedit_jg_start">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateedit_jg_end">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QDateEdit" name="dateedit_ys_start">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateedit_ys_end">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BidsAssistant.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_query</sender>
   <signal>clicked()</signal>
   <receiver>BidsAssistantClass</receiver>
   <slot>btnQueryClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1059</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_checkstatus</sender>
   <signal>clicked()</signal>
   <receiver>BidsAssistantClass</receiver>
   <slot>btnCheckStatusSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1319</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_add</sender>
   <signal>clicked()</signal>
   <receiver>BidsAssistantClass</receiver>
   <slot>btnAddCompanySlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>970</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_export</sender>
   <signal>clicked()</signal>
   <receiver>BidsAssistantClass</receiver>
   <slot>btnExportDataSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1207</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_reset</sender>
   <signal>clicked()</signal>
   <receiver>BidsAssistantClass</receiver>
   <slot>btnResetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>915</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnQueryClickedSlot()</slot>
  <slot>btnCheckStatusSlot()</slot>
  <slot>btnAddCompanySlot()</slot>
  <slot>btnExportDataSlot()</slot>
  <slot>btnResetSlot()</slot>
 </slots>
</ui>
